templates/_partials/header.tpl

```diff
@@ -0,0 +1,18 @@
+{block name="header_banner"}
+  {hook h='displayBanner'}
+{/block}
+
+{block name="header_nav"}
+  {hook h='displayNav'}
+{/block}
+
+{block name="header_logo"}
+  {* StarterTheme: Change the href for homepage link with correct protocol *}
+  <a class="logo" href="#" title="{$shop_name}">
```

Comment:
You can use `$urls.base_url` once this is merged: https://github.com/PrestaShop/PrestaShop/pull/3909